openslides_backend/action/action.py

```diff
@@ -86,9 +91,9 @@ def handle_request(self, payload: Payload, user_id: int) -> List[ActionResult]:
         # TODO: This is a fake result because in this place all actions were
         # always successful.
         self.logger.debug("Request was successful. Send response now.")
-        return [
-            ActionResult(success=True, message="Action handled successfully")
-        ] * len(payload)
+        return ActionResponse(
+            success=True, message="Actions handled successfully", results=results
```

Comment:
is `success=True` redundant? Since you have a (valid?) response, success is implied. I would expect to raise errors in error-cases and not usind `ActionResponse`, but I do not know the code well.